buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -207,9 +207,39 @@ try {
                     ghprbGhRepository = OPENJ9_REPO.substring(OPENJ9_REPO.indexOf('.com') +5, OPENJ9_REPO.indexOf('.git'))
                     ghprbActualCommit = SHAS['OPENJ9']
                     GITHUB_SERVER = OPENJ9_REPO.substring(OPENJ9_REPO.indexOf('github.'), OPENJ9_REPO.indexOf('.com') +4)
-                    PROMOTE_JOB = buildFile.build_with_slack('Promote-OpenJ9-OMR-master-to-openj9', ghprbGhRepository, ghprbActualCommit, GITHUB_SERVER,
-                                                            [string(name: 'OMR_COMMIT', value: "${SHAS['OMR']}"),
-                                                            string(name: 'OPENJ9_COMMIT', value: "${SHAS['OPENJ9']}")])
+                    PROMOTE_JOB = buildFile.build_with_slack('Promote_OMR', ghprbGhRepository, ghprbActualCommit, GITHUB_SERVER,
+                                                            [string(name: 'REPO', value: OMR_REPO),
+                                                            string(name: 'TARGET_BRANCH', value: 'openj9'),
+                                                            string(name: 'COMMIT', value: "${SHAS['OMR']}"),
+                                                            string(name: 'TAG_DESCRIPTION', value: "OpenJ9: ${SHAS['OPENJ9']}"),
+                                                            booleanParam(name: 'TAG', value: 'true')])
+                }
+            }
+            if (params.PROMOTE_OPENJDK) {
```

Comment:
There is also a parameter names `PROMOTE_OMR`. Is it required a parameter for each type of build? Could have only one parameter (string or choice) that could have different value, e.g. `PROMOTE = OMR | OPENJDK`

Reply:
Changed to switch/case